src/assets/styles/modules/blog-post.scss

```diff
@@ -0,0 +1,15 @@
+/* ================ Blog post ================ */
+.blog-post {
+  display: grid;
+  grid-column-gap: 15px;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.blog-post__date {
```

Comment:
I think we can remove this style and class completely.

Can we use a `<p>` element for the date so by default, we get `display: block`? Also, I don't think we need the margin-bottom as it will probably change when styling the theme.

Reply:
I removed it and changed to `<p>` element.